pkg/didcomm/common/service/did_comm_msg.go

```diff
@@ -48,6 +49,11 @@ func NewDIDCommMsgMap(payload []byte) (DIDCommMsgMap, error) {
 	return msg, nil
 }
 
+// NewDIDCommMsgMap returns DIDCommMsg with Header
```

Comment:
The argument to this function should be described in the function comment.

Reply:
Changed to ` NewDIDCommMsgMap converts structure(model) to DIDCommMsgMap`